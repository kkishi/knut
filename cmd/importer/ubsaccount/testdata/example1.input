Account number:;0XX0 00XX0000.X0;
IBAN:;CH20 00XX 0000 0000 0000 X;
From:;2022-01-01;
Until:;2022-12-31;
Opening balance:;0.0;
Closing balance:;900.0;
Valued in:;CHF;
Numbers of transactions in this period:;2;

Trade date;Trade time;Booking date;Value date;Currency;Debit;Credit;Individual amount;Balance;Transaction no.;Description1;Description2;Description3;Footnotes;
2022-12-31;16:58:11;2023-01-02;2022-12-31;CHF;-100.00;;;900.00;XXXXXXXXXXXXXXXX;"""Denner""";"""16787249-0 07/23; Debit card payment""";"Transaction no. XXXXXXXXXXXXXXXX";;
2022-12-21;;2022-12-21;2022-12-21;CHF;;1000.00;;1000.00;0XXXXXXXXXXXXXXX;"""Larry&Sergey Protobuf Moving Co.""";"salary payment";"""Reason for payment: Salary /; Transaction no. 0XXXXXXXXXXXXXXX""";;
