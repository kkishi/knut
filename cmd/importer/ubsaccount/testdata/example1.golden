2022-12-21 "Larry&Sergey Protobuf Moving Co."
Expenses:TBD Assets:UBS         1000 CHF

2022-12-31 "Denner"
Assets:UBS   Expenses:TBD        100 CHF

2022-12-31 balance Assets:UBS 900 CHF

